cmd/knoxite/clone.go

```diff
@@ -31,8 +31,8 @@ var (
 			if len(args) < 2 {
 				return fmt.Errorf("clone needs to know which files and/or directories to work on")
 			}
-			cloneOpts = configureStoreOpts(cmd, cloneOpts)
 
+			configureStoreOpts(cmd, cloneOpts)
```

Comment:
Careful, cloneOpts isn't a pointer. Wouldn't we lose all changes made to it inside `configureStoreOpts`?